### PR TITLE
Added / Modified Pinknode Endpoints

### PR DESCRIPTION
**Pull request by rishikarthikeyan27**
Pinknode endpoints for the following networks have been added or modified:

1) Astar
2) Shiden
3) Shibuya
4) Moonbeam
5) Moonriver
6) Moonbase Alpha
7) Polkadex
8) Khala
9) Nodle

### CODE REVIEW

File: packages/apps-config/src/endpoints/production.ts [hunk 1]
```diff
@@ -214,7 +214,8 @@ export const prodChains: EndpointOption[] = [
     text: 'Polkadex',
     providers: {
       'Polkadex Team': 'wss://mainnet.polkadex.trade',
-      OnFinality: 'wss://polkadex.api.onfinality.io/public-ws'
+      OnFinality: 'wss://polkadex.api.onfinality.io/public-ws',
+      Pinknode: 'wss://public-rpc.pinknode.io/polkadex',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/production.ts:218]: Extra trailing , making the he checks unhappy.

Comment (rishikarthikeyan27) [reply packages/apps-config/src/endpoints/production.ts]: Removed it 👍

File: packages/apps-config/src/endpoints/production.ts [hunk 2]
```diff
@@ -198,8 +198,8 @@ export const prodChains: EndpointOption[] = [
     text: 'Nodle',
     providers: {
       // Nodle: 'wss://main3.nodleprotocol.io', // https://github.com/polkadot-js/apps/issues/7652
-      OnFinality: 'wss://nodle.api.onfinality.io/public-ws'
-      // Pinknode: 'wss://rpc.pinknode.io/nodle/explorer' // https://github.com/polkadot-js/apps/issues/5721
+      OnFinality: 'wss://nodle.api.onfinality.io/public-ws',
+      Pinknode: 'wss://public-rpc.pinknode.io/nodle'
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/production.ts:202]: Duplicated under Polkadot parachains. 

The production chain is not the same as the parachain - these should be 2 different networks - https://github.com/polkadot-js/apps/runs/6806069215?check_suite_focus=true#step:3:710

**Review by jacogr — CHANGES_REQUESTED** [review]

File: packages/apps-config/src/endpoints/production.ts (hunk 2 above)
Comment (rishikarthikeyan27) [reply packages/apps-config/src/endpoints/production.ts]: Removed the endpoint from here.

**Review by jacogr — APPROVED** [review]

**Pull request merged**